### PR TITLE
BaSP-MR-21: get all, getbyid and post methods for trainers

### PR DESCRIPTION
**Pull request by LucianoStradiot**
I did the tests for the get all, by id and post methods for the trainers resource and also made a change in the trainers controller that was wrong, so that it would pass me a specific test.

### CODE REVIEW

File: src/tests/trainers.test.js [hunk 1]
```diff
@@ -1,3 +1,72 @@
...
+  });
+  test('Succesfull 200 response with an ID', async () => {
+    const res = await request(app).get('/api/trainers/64667332ecb50c522415bea5').send();
+    expect(res.status).toBe(200);
+  });
+  test('Should get a 404 response on an ID', async () => {
+    const res = await request(app).get('/api/trainers/64667332ecb50c522415bea2').send();
+    expect(res.status).toBe(404);
+  });
+  test('Should get a 400 response on an ID', async () => {
```
Comment (Eugeniagassman) [inline src/tests/trainers.test.js:46]: you could be more especific in the description

**Review by leo-balonchard — APPROVED** [review]
Looks good

**Review by fernandofernandezuy — APPROVED** [review]
Looks good to me, its ready to merge.

**Review by matiasluispeter — APPROVED** [review]
seems good!

**Pull request merged**